BlocQuery/BQProfileViewController.m

```diff
@@ -154,21 +154,27 @@ - (void)viewDidLoad
     [self.view addSubview:self.userImageView];
     self.userImageView.translatesAutoresizingMaskIntoConstraints = NO;
     
-    //enable editing buttons if you're looking at your own profile
+    //make profile editing buttons and leave them hidden by default
+    self.editUserImageButton = [UIButton buttonWithType:UIButtonTypeRoundedRect];
+    [self.editUserImageButton setTitle:@"Change Photo" forState:UIControlStateNormal];
+    [self.editUserImageButton addTarget:self action:@selector(editImageButtonPressed:) forControlEvents:UIControlEventTouchUpInside];
+    self.editUserImageButton.hidden = YES;
+    
+    self.editUserDescriptionButton = [UIButton buttonWithType:UIButtonTypeRoundedRect];
+    [self.editUserDescriptionButton setTitle:@"Edit Description" forState:UIControlStateNormal];
+    [self.editUserDescriptionButton addTarget:self action:@selector(editDescriptionButtonPressed:) forControlEvents:UIControlEventTouchUpInside];
+    self.editUserDescriptionButton.hidden = YES;
```

Comment:
We also want to make sure our UITextView doesn't accept user input... at the moment the person can tap on the description and mess with the text on a person's profile.